pkg/bootstrap/docker/up.go

```diff
@@ -583,7 +596,14 @@ func (c *ClientStartConfig) importObjects(out io.Writer, locations map[string]st
 }
 
 func (c *ClientStartConfig) openShiftImage() string {
-	return fmt.Sprintf("openshift/origin:%s", c.Tag)
+	imageTemplate := variable.NewDefaultImageTemplate()
+	imageTemplate.Format = c.Images
+	image, err := imageTemplate.Expand("")
+	if err != nil {
+		// should not happen because template was validated
+		panic(err)
+	}
+	return strings.Replace(image, "origin-:", "origin:", -1)
```

Comment:
we should probably support something in DefaultImageTemplate that trims a trailing - if component is empty.